pegasus/sites.v3/code.org/public/learn/local.haml

```diff
@@ -16,6 +16,8 @@ theme: responsive
 %script{type: "text/javascript", src: "/js/jquery.geocomplete.min.js"}
 %script{type: "text/javascript", src: "/js/jquery.colorbox-min.js"}
 %script{type: "text/javascript", src: "/js/jquery.sticky.js"}
+%link{href:'https://api.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css', rel:'stylesheet'}
+%script{src:'https://api.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.js'}
```

Comment:
Oh, should we self-host these to avoid situations where third-party sites are blocked?

Reply:
I don't think we do for our current maps but that's not a bad idea